great_expectations/expectations/metrics/util.py

```diff
@@ -359,9 +428,17 @@ def get_sqlalchemy_column_metadata(
                 sqlalchemy_engine=engine,
             )
 
+        dialect_name = execution_engine.dialect.name
+        if dialect_name == "snowflake":
```

Comment:
Do we have an enum here? Prob not since this is metrics code

Reply:
There is an enum. I'll use it.

Reply:
Done